### PR TITLE
[DO NOT MERGE]Add mixer pipeline to APL no-codec tplg

### PR DESCRIPTION
**Pull request by ranj063**
Add a mixer based low-latency pipeline and media pipeline to
the APL no-codec topology.

This PR is mainly to debug why the media pipeline that connects to the mixer cant be started.
The scheduling comp for the media pipeline is also set to SSP0 but the pipeline fails to start.

@tlauda could you please take a look? This type of pipeline is important for us to debug issues with multiple FE/multiple BE issues with dpcm.

### CODE REVIEW

**Comment by tlauda** [comment]
@ranj063 Sure, will take a look tomorrow. Have you verified if the scheduling component in the topology is set correctly for the media pipe? Your last version I have checked generated the empty sched comp and topology hasn't loaded properly.

**Comment by xiulipan** [comment]
@ranj063 
Please check on-device test result on APL/BYT nocodec. It seems it caused some regression.

**Comment by ranj063** [comment]
> @ranj063 Sure, will take a look tomorrow. Have you verified if the scheduling component in the topology is set correctly for the media pipe? Your last version I have checked generated the empty sched comp and topology hasn't loaded properly.

@tlauda yes, indeed there was tplg load error. I've fixed it now but still running into issues with starting the pipeline

File: tools/topology/sof/pipe-dai-playback.m4 [hunk 1]
```diff
@@ -20,3 +20,5 @@ W_PIPELINE(N_DAI_OUT, SCHEDULE_PERIOD, SCHEDULE_PRIORITY, SCHEDULE_FRAMES, SCHED
 
 P_GRAPH(DAI_NAME, PIPELINE_ID,
 	LIST(`		', `dapm(N_DAI_OUT, DAI_BUF)'))
+
+indir(`define', concat(`PIPELINE_PLAYBACK_SCHED_COMP_', PIPELINE_ID), N_DAI_IN)
```
Comment (tlauda) [inline tools/topology/sof/pipe-dai-playback.m4]: `N_DAI_OUT`

Comment (ranj063) [reply tools/topology/sof/pipe-dai-playback.m4]: @tlauda good catch. I've fixed it now. let me check if that fixes my problem

Comment (ranj063) [reply tools/topology/sof/pipe-dai-playback.m4]: @tlauda that fixed the problem. awesome. Thanks. I'm going to close this PR, clean up and re-submit.

**Comment by tlauda** [comment]
@ranj063 Could you also include the same changes for other platforms?

**Comment by ranj063** [comment]
> @ranj063 Could you also include the same changes for other platforms?

@tlauda what other platforms would you like at this point? I dont want to end up breaking the CI for all platforms. Can we please start with APL and then once we have the results for it, I'll gradually modify all pipelines?

**Pull request closed without merging**